usl_models/usl_models/flood_ml/dataset.py

```diff
@@ -16,6 +16,87 @@
 from usl_models.flood_ml import model
 
 
+def load_dataset(
```

Comment:
Could you describe in the docstrings when & why a caller should use `load_dataset` vs. `load_dataset_windowed` ?

Reply:
Done.